src/util/util.ts

```diff
@@ -159,6 +159,12 @@ async function generatorRunAsync(iterator: Generator<any>): Promise<any> {
   }
 }
 
+function policySortByPriority(priorityIndex: number, policy: Array<Array<string>>): void {
```

Comment:
Can you add unit test for this function?

Reply:
I remove this. There is no need to add a separate function.